### PR TITLE
localization updates

### PR DESCRIPTION
**Pull request by chappjc**
Some prominent untranslated strings stood out in https://github.com/decred/dcrdex/pull/1422#issuecomment-1014572770

- [[[Pick a server]]]
- [[[Create]]]
- intl.ID_CREATE
- intl.ID_ADD

### CODE REVIEW

File: client/webserver/locales/zh-cn.go [hunk 1]
```diff
@@ -173,4 +173,6 @@ var ZhCN = map[string]string{
 	"ready":                  "准备就绪",
 	"off":                    "关闭",
 	"Export Trades":          "导出交易",
+	"Create":                 "创",
+	"Register_loudly":        "注册!",
```
Comment (chappjc) [inline client/webserver/locales/zh-cn.go:177]: picked from other existing translations

**Review by JoeGruffins — APPROVED** [review]

File: client/webserver/locales/zh-cn.go [hunk 2]
```diff
@@ -173,4 +173,6 @@ var ZhCN = map[string]string{
 	"ready":                  "准备就绪",
 	"off":                    "关闭",
 	"Export Trades":          "导出交易",
+	"Create":                 "创",
```
Comment (martonp) [inline client/webserver/locales/zh-cn.go:176]: I think 创建 is better than just 创，but I'm not native in Chinese.

**Review by buck54321 — APPROVED** [review]

File: client/webserver/locales/pl-pl.go [hunk 3]
```diff
@@ -177,6 +177,7 @@ var PlPL = map[string]string{
 	"how_reg":                     "Jak chcesz uiścić opłatę rejestracyjną?",
 	"All markets at":              "Wszystkie rynki na",
 	"pick a different asset":      "wyierz inne aktywo",
+	"Create":                      "Utwórz",
```
Comment (chappjc) [inline client/webserver/locales/pl-pl.go:180]: @artikozel Is this one ok?

File: client/webserver/site/src/js/locales.js [hunk 4]
```diff
@@ -235,6 +242,8 @@ export const plPL = {
   [ID_NONE_SELECTED]: 'brak zaznaczenia',
   [ID_REGISTRATION_FEE_SUCCESS]: 'Płatność rejestracyjna powiodła się!',
   [ID_API_ERROR]: 'błąd API',
+  [ID_ADD]: 'Dodaj',
+  [ID_CREATE]: 'Utwórz',
```
Comment (chappjc) [inline client/webserver/site/src/js/locales.js:246]: @artikozel Are these ok?

**Review by chappjc — COMMENTED** [review]
Followed up after merging https://github.com/decred/dcrdex/pull/1422.
Please check these 2 strings, @artikozel.  Thanks!

File: client/webserver/locales/pl-pl.go [hunk 5]
```diff
@@ -177,6 +177,7 @@ var PlPL = map[string]string{
 	"how_reg":                     "Jak chcesz uiścić opłatę rejestracyjną?",
 	"All markets at":              "Wszystkie rynki na",
 	"pick a different asset":      "wyierz inne aktywo",
```
Comment (artikozel) [inline client/webserver/locales/pl-pl.go:179]: Dang it! Found a typo. It should be  "wybierz inne aktywo", Sorry!
```suggestion
	"pick a different asset":      "wybierz inne aktywo",
```

File: client/webserver/locales/pl-pl.go (hunk 3 above)
Comment (artikozel) [reply client/webserver/locales/pl-pl.go:180]: Yep! Either "Stwórz" or "Utwórz" are fine.

File: client/webserver/site/src/js/locales.js (hunk 4 above)
Comment (artikozel) [reply client/webserver/site/src/js/locales.js:246]: Yes, both are ok.

**Pull request merged**